charts/eap-xp6/templates/_helpers.tpl

```diff
@@ -4,15 +4,21 @@ eap-xp.eapBuilderImage corresponds to the imagestream for the EAP S2I Builder im
 It depends on the build.s2i.jdk version.
 */}}
 {{- define "eap-xp.eapBuilderImage" -}}
+{{- if eq .Values.build.s2i.jdk "17"  -}}
 {{ .Values.build.s2i.jdk17.builderImage}}
+{{- else -}}
+{{ .Values.build.s2i.jdk21.builderImage}}
 {{- end -}}
 
 {{/*
 eap-xp.eapRuntimeImage corresponds to the imagestream for the EAP S2I Runtime image.
 It depends on the build.s2i.jdkVersion.
 */}}
 {{- define "eap-xp.eapRuntimeImage" -}}
+{{- if eq .Values.build.s2i.jdk "17"  -}}
 {{ .Values.build.s2i.jdk17.runtimeImage}}
+{{- else -}}
+{{ .Values.build.s2i.jdk21.builderImage}}
 {{- end -}}
```

Comment:
same here?

Reply:
@mchoma yes, I cherry picked your commit to fix both places. thanks!